save-cloud-charts/save-cloud/templates/backend-cosv.yaml

```diff
@@ -131,9 +133,11 @@ spec:
             secretName: s3-secrets
         - name: migrations-data
           emptyDir: {}
+        {{ if .Values.gateway.initCerts.enabled }}
```

Comment:
why we use gateway property in backend?

Reply:
because if gateway need to enable initCerts, so do backend and i reuse this value in backend and cosv backend

Reply:
can we move then this property to upper level? `.Values.initCerts.enabled` instead of `.Values.gateway.initCerts.enabled`

Reply:
done as suggested